tests/spyre_util.py

```diff
@@ -537,3 +537,10 @@ def create_text_prompt(model: str, min_tokens: int, max_tokens: int) -> str:
     assert min_tokens < len(tokenizer.encode(prompt)) < max_tokens
 
     return prompt
+
+
+def skip_unsupported_tp_size(size: int):
```

Comment:
Just a nit but something more verbose like `skip_if_insufficient_cards` might be more accurate/readable.